app/views/confirm.scala.html

```diff
@@ -48,14 +48,14 @@ <h1 class="govuk-panel__title">@messages("label.confirm.heading")</h1>
     <p class="govuk-body">
         <a href="@routes.ApplicationController.print" class="print-link govuk-link" target="_blank">@messages("label.button.download")</a>
     </p>
-    
+
     <h2 class="govuk-heading-m">@messages("label.whatNext")</h2>
 
    <p class="govuk-body">@messages("confirm.copy.list.1")</p>
    <p class="govuk-body">@messages("confirm.copy.list.2")</p>
 
     <p class="govuk-body">
-      <a href="@controllers.feedback.routes.FeedbackController.feedback.url">@messages("confirm.link.feedback")</a>
+        <a class="govuk-link" href="@controllers.feedback.routes.FeedbackController.feedback.url?journey=normal-journey">@messages("confirm.link.feedback")</a>
```

Comment:
Why we need to add additional parameter to URL?

Reply:
Actually we don't need that. Good spot.